ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -94,6 +105,9 @@ def check(self, instance):
                     self.service_check(self.SFTP_SERVICE_CHECK_NAME, status, tags=tags, message=exception_message)
                 raise
 
+            _, _, stderr = client.exec_command('ssh -V')
```

Comment:
Since the check is about checking the connection to a remote server. Should we set the version to the remote version instead? 

https://github.com/bitprophet/ssh/blob/master/ssh/transport.py#L1680-L1681